.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build
+on:
+  push:
+    branches:
```

Comment:
This is a good first step. But would recommend adding exceptions for builds/tests to run on specific paths: i.e https://github.com/SolaceDev/maas-billing-v2/blob/8961d14770a4113486267a95cea288e51009ff3e/.github/workflows/release.yml#L5

Reply:
Or only run build/test when there's a change in the service folder. Example: https://github.com/SolaceDev/maas-billing-v2/blob/8961d14770a4113486267a95cea288e51009ff3e/.github/workflows/build-helm-chart.yml#L10

Reply:
That's a good idea, so we dont need to run tests on for exampl egithub actions changes.

Reply:
after discussing it with @amanek-solace we decided to add this later.

Reply:
lol that's a two line change: 
```
  paths:
      - 'service/**'
```